sdk/azidentity/go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/Azure/azure-sdk-for-go/sdk/azidentity
 
 go 1.16
 
+replace github.com/Azure/azure-sdk-for-go/sdk/internal => ../internal
```

Comment:
Does this stick around until this merges to main?

Reply:
Yes, I'll have to do a release of internal first then they'll all be removed before the next release cycle.